susemanager-utils/testing/automation/spacewalk-pylint.sh

```diff
@@ -158,7 +153,7 @@ INITIAL_CMD="/manager/susemanager-utils/testing/automation/initial-objects.sh"
 PYLINT_CMD="mkdir -p /manager/reports; cd /manager/; pylint --disable=E0203,E0611,E1101,E1102,C0111,I0011,R0801 --ignore=test --output-format=parseable --rcfile /manager/spacewalk/pylint/spacewalk-pylint.rc --reports=y --msg-template=\"{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}\""
 CHOWN_CMD="/manager/susemanager-utils/testing/automation/chown-objects.sh $(id -u) $(id -g)"
 
-docker pull $REGISTRY/$PYLINT_CONTAINER
```

Comment:
If we still have `$PYLINT_CONTAINER` at `$HERE/VERSION, maybe we should remove it? And if we have a Docker file for it, we can remove it as well. But why are we using the PostgreSQL image for this?

Reply:
I checked the VERSION and I could not find a PYLINT_CONTAINER there. Did I overlook something?

Reply:
So this was basically broken :-?

Very weird.

Reply:
@mcalmer now that I think about this, I think the reason this did not fail is that for master we use the Head containers.

Not sure about the PRs.

I guess we better get a review from @jordimassaguerpla for this.

Reply:
We don run pylint in the Pull Requests.

Reply:
In head, I just realized the pylint tests are broken with:

ImportError: Unable to find module for /manager/usix/common/usix.py in /manager/susemanager,

Reply:
see for example https://ci.suse.de/view/Manager/view/Manager-Head/job/manager-Head-dev-susemanager-python-pylint/3762/console

Reply:
@mcalmer Do the spacewalk-pylint.sh work with your changes? If so, we will fix it with your PR ;)

Reply:
Yes, it will work. I tested this on top of https://github.com/uyuni-project/uyuni/pull/3761 and at least the tests started.
```
docker pull $REGISTRY/$PYLINT_CONTAINER
```
worked by **pure** luck. Because we pull that, but we run with `$REGISTRY/$PGSQL_CONTAINER` :-)

Reply:
> We don run pylint in the Pull Requests.

Uh-oh, then we should, FMPOV.

Reply:
> Yes, it will work. I tested this on top of #3761 and at least the tests started.
> 
> ```
> docker pull $REGISTRY/$PYLINT_CONTAINER
> ```
> 
> worked by **pure** luck. Because we pull that, but we run with `$REGISTRY/$PGSQL_CONTAINER` :-)

Keep in mind tests will work, but we still have something wrong with the tests themselves:
```
$ susemanager-utils/testing/automation/spacewalk-pylint.sh
Problem importing module base.py: cannot import name 'get_node_last_lineno'

** COLLECTED WARNINGS **
Failed to open memory buffer (/dev/mem): No such file or directory
No SMBIOS nor DMI entry point found, sorry.
** END OF WARNINGS **

*********** ADJUSTING PERMISSIONS FOR OBJECTS GENERATED INSIDE THE CONTAINER TO 1000:100 ***********
$ echo $?
1
```
But FMPOV this is something for the python people :-)

Reply:
Closing this.